network-store-iidm-impl/src/main/java/com/powsybl/network/store/iidm/impl/AbstractTapChangerStepsReplacer.java

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright (c) 2023, All partners of the iTesla project (http://www.itesla-project.eu/consortium)
```

Comment:
I know that the PR was created in 2023 but the code will be added in 2024
```suggestion
 * Copyright (c) 2024, All partners of the iTesla project (http://www.itesla-project.eu/consortium)
```

Reply:
Yeah I don't know the rule in this case, I created the file in 2023 so well... I change it anyway